scheduler-actor-reminders/README.md

```diff
@@ -0,0 +1,51 @@
+# Scheduler Actor Reminders
+
+## Overview
+This project tests the Dapr Scheduler for handling Actor Reminders using an example `PlayerActor` actor. The example
+simulates a game session where a player's health increases or decays periodically, managed through Dapr Actor reminders.
+
+## Project Structure
+The `client` directory implements the client code that interacts with `PlayerActor`, specifically setting up reminders,
+monitoring health, and handling shutdown. It invokes actor methods, but does not manage the actor lifecycle.
+
+The `server` directory implements the `PlayerActor` server code, which manages a game session for a player, including
+health-based reminders. This code defines the actor lifecycle and its reminder-based state changes.
+
+## Reminders
+Two reminders manage the actor's health:
+- `healthReminder`:
+    - Increases the player's health if it is below full health.
+- `healthDecayReminder`:
+    - Decreases the player's health periodically, simulating a natural decay over time.
+
+When the player's health reaches 0, the client unregisters the reminders, revives the player, and restarts the reminders.
+
+This tests the Scheduler for the underlying storage for Actor Reminders.
+
+## How To Run the Code:
```

Comment:
I'm able to run it as-is using the instructions from my readme and do not see that error. Did you try the exact commands from my readme? Im also using v1.14.4

I have seen that error before- outside of the context of my PR here -in the wf scheduler e2e PR in CI, but am not able to repro the issue you are seeing and not familiar enough actors to know what the issue is off the top of my head.

Reply:
First, I would try the commands noted in the readme since thats exactly what I've tested and what works for me. If that doesn't work you you, then I would try leaving `--placement-host-address` unset and see if that changes things. I noticed you are running placement in HA mode from your command.

Reply:
Yeah, I did run the original commands as they are in the read me, and I got this error:
```
== APP == 2024/11/20 17:25:02 error invoking actor method GetUser: error invoking binding playerActorType/player-1: rpc error: code = Internal desc = the state store is not configured to use the actor runtime. Have you set the - name: actorStateStore value: "true" in your state store component file?
```

Then I removed the ` --app-port 3008` argument for the client and got the error I shared above. Really weird. I'll look into it some more.